docusaurus.config.js

```diff
@@ -41,11 +41,26 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
           editUrl: 'https://github.com/apache/incubator-linkis-website/edit/dev/',
           versions: {
             current: {
-              path: '1.1.2',
+              path: '1.1.1',
             },
-            '1.1.1': {
-              path: 'latest',
```

Comment:
the current docs version is Next-1.1.2 (WIP)
1.1.1 has been released, docs latest version is 1.1.1.

Reply:
resolved